api/app/db/models/advisory.py

```diff
@@ -54,8 +44,30 @@ class Shape(Base):
     # An area is uniquely identified, e.g. a zone has a number, so does a fire.
     source_identifier = Column(String, nullable=False, index=True)
     shape_type = Column(Integer, ForeignKey('advisory_shape_types.id'), nullable=False, index=True)
-    geom = Column(Geometry('MULTIPOLYGON', spatial_index=False), nullable=False)
+    geom = Column(Geometry('MULTIPOLYGON', spatial_index=False, srid=NAD83_BC_ALBERS), nullable=False)
 
 
 # Explict creation of index due to issue with alembic + geoalchemy.
 Index('idx_advisory_areas_geom', Shape.geom, postgresql_using='gist')
+
+
+class ClassifiedHfi(Base):
+    """ HFI classified into different groups.
+    NOTE: In actual fact, forecasts and actuals can be run multiple times per day,
+    but we only care about the most recent one, so we only store the date, not the timesamp.
+    """
+    __tablename__ = 'advisory_classified_hfi'
+    __table_args__ = (
+        {'comment': 'HFI classification for some forecast/advisory run on some day, for some date'}
+    )
+    id = Column(Integer, primary_key=True, index=True)
+    # TODO: we could do this better!
```

Comment:
Maybe briefly explain *how* this could be done better?

Reply:
I've added a comment. It's basically around not using a string. We could fix this now if you want? I'm feeling a bit non-committal at the moment, but normalizing the database is probably a good idea. Should I do it?

Reply:
Going to normalize this...

Reply:
Normalized.